.github/workflows/tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     steps:
       - name: Checkout source
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.5.2
```

Comment:
Why not `v3`, like it advises here?
https://github.com/marketplace/actions/checkout
Do we have to be explicit? Not a blocking comment, just curious.

Reply:
I was following what we have in dask/dask but we can just do v3 and see what happens